libindy/Cargo.toml

```diff
@@ -38,17 +38,17 @@ sodium_static = []
 indy-crypto = { version = "=0.4.1", optional = true }
 int_traits = { version = "0.1.1", optional = true }
 digest = "0.6.2"
-env_logger = "0.4.2"
+env_logger = "0.5.10"
 errno = "0.2.3"
 etcommon-rlp = "0.2.3"
 generic-array = "0.8.3"
 hex = "0.2.0"
 libc = "0.2.21"
-log = "0.3.7"
+log = "0.4.1"
 openssl = { version = "=0.9.24", optional = true }
 owning_ref = "0.3.3"
 rand = "0.3"
-rusqlite = "0.13.0"
+rusqlite = { version = "0.13.0", features=["bundled"] }
```

Comment:
As i understand now we start static linking with sqlite on all platforms.

Reply:
On Android, we don't have SQLite binaries which we provide while building. Also, I can't recall any flag which suggests that we are doing static linking with SQLite.

Reply:
The problem is that we don't want to force static linking on all platforms. Only for android and iOS